apps/src/templates/sectionAssessments/FreeResponseDetailsDialog.story.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import FreeResponseDetailsDialog from '@cdo/apps/templates/sectionAssessments/FreeResponseDetailsDialog';
+
+export default storybook => {
+  return storybook
+    .storiesOf('Dialogs/FreeResponseDetailsDialog', module)
+    .addStoryTable([
+      {
+        name: 'FreeResponseDetailsDialog',
+        description: 'Detail view of a free response question',
+        story: () => (
+          <FreeResponseDetailsDialog
+            isDialogOpen={true}
+            closeDialog={() => {}}
+            questionText={"Hello world. I display markdown questions."}
```

Comment:
tiny nit: it would be nice if this example string used some markdown syntax

Reply:
Will follow up in a separate PR.